_source/logzio_collections/_log-sources/platform-service/zipkin.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Ship Zipkin traces
+logo:
+  logofile: zipkin.svg
+  orientation: vertical
+open-source:
+  - title: Zipkin-Logz.io Trace Storage
+    github-repo: zipkin-logzio
+shipping-summary:
+  data-source: Zipkin traces
+contributors:
+  - imnotashrimp
+---
+
+Zipkin-Logz.io Trace Storage is a storage option for Zipkin distributed traces on your Logz.io account.
+It can function as both a collector and a span store.
+
+<div class="info-box note">
+  This integration requires Logz.io API access.
+  The Logz.io API is available for all Enterprise accounts.
+  If you're on a Pro account, reach out to your account manager or the <a class="intercom-launch" href="mailto:sales@logz.io">Sales team</a> to request API access.
+</div>
+
+### Limitations
+
+When you use the Zipkin UI to find traces stored in Logz.io, there are a couple limitations.
+For most users, these won't be an issue, but they're still good to know:
+
+* **Lookback** must be 2 days or less
+* **Limit** must be 1000 traces or less
+
+## To integrate Zipkin server and Logz.io
+
+{: .tasklist .firstline-headline}
+1. Download Zipkin server and Zipkin-Logz.io Trace Storage
+
+    Download [Zipkin server](https://search.maven.org/remote_content?g=io.zipkin.java&a=zipkin-server&v=LATEST&c=exec).
+
+    ```shell
+    wget -O zipkin.jar 'https://search.maven.org/remote_content?g=io.zipkin.java&a=zipkin-server&v=LATEST&c=exec'
+    ```
+
+    Download the [Zipkin-Logz.io Trace Storage](https://github.com/logzio/zipkin-logzio/releases) jar to the same directory.
+
+2. Run Zipkin server with the Logz.io extension
+
+    You can configure the Logz.io extension with shell variables or environment variables.
+
+    For a complete list of options, see the parameters below the code block.👇
+
+    ```bash
+    STORAGE_TYPE=logzio \
+    LOGZIO_ACCOUNT_TOKEN=<ACCOUNT-TOKEN> \
+    LOGZIO_LISTENER_HOST=<LISTENER-URL> \
+    LOGZIO_API_TOKEN=<API-TOKEN> \
+    LOGZIO_API_HOST=<API-URL> \
+    java -Dloader.path='zipkin-logzio.jar,zipkin-logzio.jar!lib' -Dspring.profiles.active=logzio -cp zipkin.jar org.springframework.boot.loader.PropertiesLauncher
+    ```
+
+    <div class="info-box tip">
+      You can optionally run two Zipkin-Logzio Trace Storage instances if you want to separate shipping and reading of your traces.
+
+      If you do, then the required fields change a bit from what's shown in the Parameters list:
+
+      * The **shipping instance** requires `STORAGE_TYPE=logzio`, `LOGZIO_ACCOUNT_TOKEN`, and `LOGZIO_LISTENER_HOST` (if you're not shipping to the default listener).
+      * The **reading instance** requires `STORAGE_TYPE=logzio`, `LOGZIO_API_TOKEN`, and `LOGZIO_API_HOST`.
```

Comment:
@imnotashrimp let's keep consistency of the required parameters, either both LOGZIO_API_HOST and LOGZIO_LISTENER_HOST should be required or neither of them..